fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/DimensionRowSerializer.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.data.dimension;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Serialization strategy for DimensionRow.
+ */
+public class DimensionRowSerializer extends JsonSerializer<DimensionRow> {
+
+    @Override
+    public void serialize(DimensionRow value, JsonGenerator gen, SerializerProvider serializers) throws IOException {
```

Comment:
My total bad.  I'm not sure if I was looking at the wrong javadoc, but I see in 2.2 for the fasterxml code, these are thrown.
I swear that I saw a version somewhere where there were not explicit throws.